frontend/app/src/components/helper/Card.vue

```diff
@@ -26,27 +26,21 @@ const props = withDefaults(
 );
 
 const { contained } = toRefs(props);
-const body = ref<HTMLDivElement | null>(null);
-const actions = ref<HTMLDivElement | null>(null);
-const top = ref(206);
+const titleRef = ref<HTMLDivElement | null>(null);
+const subTitleRef = ref<HTMLDivElement | null>(null);
+const actionsRef = ref<HTMLDivElement | null>(null);
```

Comment:
small nit maybe we can just do `ref<HTMLDivElement>()` which will result into `Ref<HTMLDivElement | undefined>` being the type and we get rid of the needed initialization.

Most of this initialization to null was a problem with normal options properties anyway before `Ref` where undefined was not reactive, which is not a problem anymore if with refs.